src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -9932,6 +9930,44 @@ public void LangVersionFlag()
                 VerifyQuotedValid("langversion", "2", LanguageVersion.CSharp2, x => x.ParseOptions.LanguageVersion);
             }
         }
+
+        [Fact]
+        [WorkItem(23525, "https://github.com/dotnet/roslyn/issues/23525")]
+        public void InvalidPathCharacterInPathMap()
+        {
+            string filePath = Temp.CreateFile().WriteAllText("").Path;
+            var compiler = new MockCSharpCompiler(null, _baseDirectory, new[]
```

Comment:
>MockCSharpCompiler [](start = 31, length = 18)

Pass "/preferreduilang:en" switch otherwise the test is likely to fail on non-english machine